### PR TITLE
MV3 background event pages

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary
MV3 background event pages related changes including:

- note about non-persistent background page support change in Firefox 101, 
- flagged extension.lastError, extension.getBackgroundPage, and runtime.getBackgroundPage as deprecated

#### Test results and supporting details
Tested and passed `npm test`

### CODE REVIEW

File: webextensions/api/extension.json [hunk 1]
```diff
@@ -82,6 +82,11 @@
               "safari_ios": {
                 "version_added": "15"
               }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": false,
+              "deprecated": true
```
Comment (mixedpuppy) [inline webextensions/api/extension.json:89]: getBackgroundPage is not deprecated

File: webextensions/api/runtime.json [hunk 2]
```diff
@@ -482,6 +482,11 @@
               "safari_ios": {
                 "version_added": "15"
               }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": false,
+              "deprecated": true
```
Comment (mixedpuppy) [inline webextensions/api/runtime.json:489]: getBackgroundPage is not deprecated

File: webextensions/manifest/background.json [hunk 3]
```diff
@@ -61,17 +61,15 @@
               },
               "firefox": {
                 "version_added": "48",
-                "partial_implementation": true,
                 "notes": [
-                  "Only persistent pages are supported.",
+                  "Before version 101, only persistent pages are supported.",
```
Comment (mixedpuppy) [inline webextensions/manifest/background.json:65]: persistent pages are not yet supported.  they are pref'd off and available to developers to pref on for testing.  we might want to reword that, and have tickets to return to these items later.  maybe something like:

"Only persistent pages are supported.  As of 101 addon developers may turn on event pages for testing purposes"

**Review by mixedpuppy — CHANGES_REQUESTED** [review]

File: webextensions/api/extension.json (hunk 1 above)
Comment (rebloor) [reply webextensions/api/extension.json]: Sorry, must've misunderstood your comment, reverted

File: webextensions/api/runtime.json (hunk 2 above)
Comment (rebloor) [reply webextensions/api/runtime.json]: Ditto here too

File: webextensions/manifest/background.json (hunk 3 above)
Comment (rebloor) [reply webextensions/manifest/background.json:65]: Using the MV3 preference I assume? e.g. "Only persistent pages are supported. As of version 101, extension developers can turn on event pages for testing purposes using the `extensions.manifestV3.enabled` preference."

Comment (mixedpuppy) [reply webextensions/manifest/background.json:65]: They can also turn on event pages for MV2 using `extensions.eventPages.enabled` and that will be in the blog posts.

**Review by mixedpuppy — APPROVED** [review]
r+ with mention of MV2 support pref

**Pull request merged**